db/migrate/20231119055722_add_status_to_webhooks.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToWebhooks < ActiveRecord::Migration[6.1]
+  def change
+    add_column :webhooks, :status, :string
```

Comment:
Acho que pode ficar legal ter um valor padrão

```suggestion
    add_column :webhooks, :status, :string, default: 'active'
```